domain/domain.go

```diff
@@ -356,12 +358,15 @@ type etcdBackend interface {
 }
 
 // NewDomain creates a new domain. Should not create multiple domains for the same store.
-func NewDomain(store kv.Storage, lease time.Duration) (d *Domain, err error) {
+func NewDomain(store kv.Storage, lease time.Duration, factory pools.Factory) (d *Domain, err error) {
+	minCapacity := 1               // minCapacity for the sysSessionPool size
+	maxCapacity := 200             // maxCapacity for the sysSessionPool size
```

Comment:
OK, I'll enlarge it to 500.
ExecuteRestrictedSQL use session from this pool, it should not be too much, I think.